guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -16,18 +16,21 @@ For more information, see {ReleaseNotesDocURL}[Release notes].
 endif::[]
 * {Project} {ProjectVersion} or {SmartProxy} {ProjectVersion} running on {EL} 8.
 ifdef::satellite[]
-* For {SmartProxy} upgrade, enable and synchronize the following repositories to {ProjectServer}:
-+
-Red Hat Enterprise Linux 9 for x86_64 - BaseOS (RPMs):: `{RepoRHEL9BaseOS}`
-+
-Red Hat Enterprise Linux 9 for x86_64 - AppStream (RPMs):: `{RepoRHEL9AppStream}`
-+
-Red Hat Satellite Capsule {ProjectVersion} for RHEL 9 x86_64 RPMs:: `{RepoRHEL9ServerSatelliteCapsuleProjectVersion}`
-+
-Red Hat Satellite Maintenance {ProjectVersion} for RHEL 9 x86_64 RPMs:: `{RepoRHEL9ServerSatelliteMaintenanceProjectVersion}`
-
-+
-Add the repositories to the lifecycle environment and content view that is attached to your {SmartProxyServer}.
+* If you are upgrading a {SmartProxy}, enable and synchronize the following repositories to {ProjectServer}, and add them to the lifecycle environment and content view that is attached to your {SmartProxyServer}:
+** *Red Hat Enterprise Linux 9 for x86_64 - BaseOS (RPMs)*:
+*** `{RepoRHEL9BaseOS} 9`
+*** `{RepoRHEL9BaseOS} 9._Y_` where _Y_ represents the latest released major version.
```

Comment:
```suggestion
*** `{RepoRHEL9BaseOS} 9._Y_` where _Y_ represents the latest supported minor version.
```
Y is minor (for RHEL, at least).
whether we want to say "released" or "supported", up to you :)

Reply:
My bad. Addressed.